app/controllers/admin/locations_controller.rb

```diff
@@ -18,11 +18,12 @@ def create
     if @location.save
       redirect_to admin_locations_path
     else
-      flash[:error] = "Something went wrong :("
+      flash.now[:error] = "Something went wrong :("
       render 'new'
     end
   end
 
+  # FIXME: This should probably have error handling like create
```

Comment:
Should I make a card for this? I'm not sure how your flow works. Sorry if I asked this before, heh.

Reply:
@rsl good catch.  I am happy to add cards, particularly since I can adjust the priority in the backlog.  I added one for this one (which also includes two other controllers where we have left this out): https://trello.com/c/u7USxYQZ/137-add-error-handling-in-update-for-judges-lawyers-and-locations-controllers
Thanks!